workflow/util/util.go

```diff
@@ -219,7 +220,12 @@ func SubmitWorkflow(wfIf v1alpha1.WorkflowInterface, wfClientset wfclientset.Int
 			}
 
 			for k, v := range yamlParams {
-				value := fmt.Sprintf("%s", v)
+				// We get quoted strings from the yaml file.
+				value, err := strconv.Unquote(fmt.Sprintf("%s", v))
```

Comment:
Very minor suggestion:
```suggestion
				value, err := strconv.Unquote(string(v))
```